docs/modules/ROOT/attachments/ica/context/index.json

```diff
@@ -1,5 +1,35 @@
 {
   "@context": {
-    "@vocab": "https://www.w3.org/ns/credentials/examples#"
+    "cawg": "https://cawg.io/terms#",
```

Comment:
Can we move this URL under `cawg.io/identity`? That will make it easier to map the source docs to how we host them.